src/tests/cptests/Makefile

```diff
@@ -38,7 +38,17 @@ clean:
 fuzz_parent_test_objects = $(foreach obj,$(notdir $(parent_test_objects)),fuzz-$(obj))
 fuzz_objects = $(foreach obj,$(parent_objs),fuzz-$(obj))
 
-fuzz: prepare-incdir fuzz.cc $(fuzz_parent_test_objects) $(fuzz_objects)
+# Create mconfig.h for fuzzing. because fuzz target not runned by top level make; we need to create mconfig.h manually.
+../../../mconfig:
+	$(MAKE) -C ../../../ mconfig
+
+../../../build/includes/mconfig.h: ../../../mconfig
+	$(MAKE) -C ../../../build all
+
+include/mconfig.h: ../../../build/includes/mconfig.h
+	/usr/bin/env mv ../../../build/includes/mconfig.h includes/mconfig.h
```

Comment:
Rather than move the generated mconfig.h, could you just add the ../../../build/includes directory to the include path when compiling (i.e. add `-I../../../build/includes` to the command line for the compilation targets) and change the dependency in `fuzz` from `includes/mconfig.h` to `../../../build/includes/mconfig.h`. Then you can remove this `include/mconfig.h` target altogether.

Reply:
Waiting for testing that...